content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,425 @@
+you should determine if the request should be broader and include more
+possible codes to account for different ways data might be coded or work
+with us to determine the best way to return the information you need.
+
+For example, lead can be performed on blood, capillary blood, or venous blood.
+
+- **Hospital A** uses a proprietary code for lead which maps to LOINC code `5671-3` - `(Lead [Mass/volume] in Blood)`
+- **Hospital B** uses a proprietary code for lead which maps to LOINC code `77307-71` - `(Lead [Mass/volume] in Venous blood)`
+
+If you are using a specific code to query for lead and use code
+`5671-3`, then your app would only return results when installed at
+**Hospital A**.  Your app would return nothing if installed at
+**Hospital B**. You need to ensure that all codes are accounted for in
+order for the app to work at all sites.
+
+Another scenario is where hospitals have different tests and codes setup
+for different methodologies. An example is when a separate test and code
+is created to document blood lead collected by venipuncture versus a
+capillary draw, which is a common collection method for children. On
+occasion, client proprietary codes may not map directly to any LOINC
+code. For special cases where we discover a proprietary code that does
+not map cleanly to any standard, we will work with you to figure out the
+best option for that mapping during deployment.
+
+For each of these scenarios, it is one thing to validate against our
+sandbox to adhere to the FHIR specification, but we also need to
+validate your app at each client site to ensure that there aren’t any
+hospital-specific considerations to be aware of. We will do this when we
+connect your app to a hospital domain. We will work with you to resolve
+any of these hospital-specific considerations.
+
+The above examples use lead, but similar considerations could exist for
+virtually any lab or measurement where concept mapping and specific
+LOINC codes are used.  If your app uses specific LOINC codes to query
+for information,
+[contact us](https://groups.google.com/d/forum/cerner-fhir-developers)
+and we will help you work through any considerations that might exist.
+
+### Leverage Query Parameters
+
+We have seen a lot of instances of developers performing post-query
+filtering rather than leveraging the
+[query parameters](https://www.hl7.org/fhir/dstu2/search.html#table)
+defined by the API.  We recommend using the available query parameters, such as
+[this example for the Patient resource](http://fhir.cerner.com/millennium/dstu2/individuals/patient/#parameters).
+Query parameters make the dataset more predictable, speed up your searches, and
+are more reliable than relying on post-query filtering.
+
+### Review Filtering Options
+
+Using Query Parameters is what we recommend instead of post-query filtering,
+but if you are filtering, here is one consideration to be aware of. In the
```

Comment:
Is this trying to say: "Using query parameters is recommended over post-query filtering (when possible). Regardless of the method used to filter results, ensure that your application handles resources that have been marked in error or invalid using the appropriate status field(s) for each resource." ? In error is only one example, there may be other values that the app, I think that's what the second paragraph is getting at.